src/frontend/screens/Settings/ProjectConfig/index.tsx

```diff
@@ -62,8 +63,8 @@ export const ProjectConfig = () => {
   const [role] = React.useState<"participant" | "coordinator">("participant");
   const [status, setStatus] = React.useState<Status>("idle");
   const [config, { replace: replaceConfig }] = React.useContext(ConfigContext);
-  const [{ experiments }] = React.useContext(SettingsContext);
 
+  const { onboarding } = devExperiments;
```

Comment:
can extract this value outside of the the component since it's a stable constant